tutorials/blockchains/custom-blockchains/build-a-blockchain/simulate-a-network.md

```diff
@@ -0,0 +1,200 @@
+
+- Node identity:
+
+    ```plain
+    2024-09-10 08:35:43 🏷  Local node identity is: 12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
+    ```
+
+    This string uniquely identifies the node. It's determined by the `--node-key` used to start the node with the Alice account. Use this identifier when connecting additional nodes to the network.
+
+- Network status:
+
+    ```plain
+    2024-09-10 08:35:48 💤 Idle (0 peers), best: #0 (0x850f…951f), finalized #0 (0x850f…951f), ⬇ 0 ⬆ 0
+    ```
+
+    This message indicates that:
+
+    - No other nodes are currently in the network
+    - No blocks are being produced
+    - Block production will commence once another node joins the network
+
+## Add a Second Node to the Network
+
+After successfully running the first node with the Alice account keys, you can expand the network by adding a second node using the Bob account. This process involves connecting to the existing network using the running node as a reference point. The commands are similar to those used for the first node, with some key differences to ensure proper network integration.
+
+To add a node to the running blockchain:
+
+1. Open a new terminal shell on your computer
+
+2. Navigate to the root directory where you compiled the Polkadot SDK Solochain Template
+
+3. Clear any existing chain data for the new node:
+
+    ```bash
+    ./target/release/solochain-template-node purge-chain --base-path /tmp/bob --chain local -y
+    ```
+
+    !!!note
+        The `-y` flag automatically confirms the operation without prompting.
+
+4. Start the second local blockchain node using the Bob account:
+    ```bash
+    ./target/release/solochain-template-node \
+    --base-path /tmp/bob \
+    --chain local \
+    --bob \
+    --port 30334 \
+    --rpc-port 9946 \
+    --node-key 0000000000000000000000000000000000000000000000000000000000000002 \
+    --validator \
+    --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
+    ```
+
+    Key differences in this command:
+    
+    1. Unique paths and ports - to avoid conflicts on the same machine, different values are used for:
+        - `--base-path` - set to `/tmp/bob`
+        - `--port` - set to `30334`
+        - `--rpc-port` - set to `9946`
+
+    2. Bootnode specification - the `--bootnodes` option is crucial for network discovery:
+        - Format - `/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp`
+        - Components:
+            - `ip4` - indicates IPv4 format
+            - `127.0.0.1` - IP address of the running node (localhost in this case)
+            - `tcp` - specifies TCP for peer-to-peer communication
+            - `30333` - port number for peer-to-peer TCP traffic
+            - `12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp` - unique identifier of the Alice node
```

Comment:
```suggestion
    - **Unique paths and ports** - to avoid conflicts on the same machine, different values are used for:
        - **`--base-path`** - set to `/tmp/bob`
        - **`--port`** - set to `30334`
        - **`--rpc-port`** - set to `9946`

    - **Bootnode specification** - the `--bootnodes` option is crucial for network discovery:
        - **Format** - `/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp`
        - **Components**:
            - **`ip4`** - indicates IPv4 format
            - **`127.0.0.1`** - IP address of the running node (localhost in this case)
            - **`tcp`** - specifies TCP for peer-to-peer communication
            - **`30333`** - port number for peer-to-peer TCP traffic
            - **`12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp`** - unique identifier of the Alice node
```